packages/admin-panel/src/autocomplete/ReduxAutocomplete.jsx

```diff
@@ -37,12 +39,27 @@ const ReduxAutocompleteComponent = React.memo(
     placeholder,
     helperText,
   }) => {
+    const [hasSetInitialSelection, setHasSetInitialSelection] = useState(false);
```

Comment:
There might be a better way to ensure this only is done once, but this is what first came to mind...